src/main/java/frc/robot/RobotContainer.java

```diff
@@ -95,37 +95,61 @@ public RobotContainer() {
           {
             System.out.println("Drivebase enabled");
             driveBaseSubsystem = new DriveBaseSubsystem(driverJoystick);
-            //driveBaseTeleopCommand = new DriveBaseTeleopCommand(driveBaseSubsystem);/TODO: can't assign this, command is final!d
-            driveBaseSubsystem.setDefaultCommand(driveBaseTeleopCommand);
             break;
           }
           case "CDSSubsystem": 
           {
             System.out.println("CDS enabled");
-            CDSSubsystem = new CDSSubsystem();
-            CDSForwardCommand = new CDSForwardCommand(CDSSubsystem,shooterSubsystem);
-            CDSReverseCommand = new CDSReverseCommand(CDSSubsystem,shooterSubsystem);
+//            CDSSubsystem = new CDSSubsystem();
             break;
           }
           case "IntakeSubsystem":
           {
             System.out.println("Intake enabled");
             intakeSubsystem = new IntakeSubsystem(); 
-            intakeForwardCommand = new IntakeForwardCommand(intakeSubsystem);
-            intakeReverseCommand = new IntakeReverseCommand(intakeSubsystem);
             break;
           }
           case "ShooterSubsystem":
           {
             System.out.println("Shooter enabled");
             shooterSubsystem = new ShooterSubsystem();
-            shooterPrime = new ShooterPrime(shooterSubsystem, limelightSubsystem,CDSSubsystem);
             break;
           }
           case "LimelightSubsystem":
           {
             System.out.println("Limelight enabled");
             limelightSubsystem = new LimelightSubsystem();
+            break;
+          }
+
+        }
+        switch (sub.toString()) {
+          case "DriveBaseSubsystem": 
+          {
+            driveBaseTeleopCommand = new DriveBaseTeleopCommand(driveBaseSubsystem);
+            driveBaseSubsystem.setDefaultCommand(driveBaseTeleopCommand);
+            break;
+          }
+          case "CDSSubsystem": 
+          {
+            //CDSForwardCommand = new CDSForwardCommand(CDSSubsystem, shooterSubsystem);
+            //CDSReverseCommand = new CDSReverseCommand(CDSSubsystem, shooterSubsystem);
+            break;
+          }
+          case "IntakeSubsystem":
+          {
+            intakeForwardCommand = new IntakeForwardCommand(intakeSubsystem);
+            intakeReverseCommand = new IntakeReverseCommand(intakeSubsystem);
+            beamBreakCommand = new BeamBreakCommand(intakeSubsystem);
+            break;
+          }
+          case "ShooterSubsystem":
+          {
+            shooterPrime = new ShooterPrime(shooterSubsystem, limelightSubsystem, CDSSubsystem);
+            break;
+          }
+          case "LimelightSubsystem":
+          {
```

Comment:
This is outdated, see Shooter Test branch for the new checks that look for null subsystems before instantiating the commands inside configureButtonBindings